app/docs/0.10.x/secure-admin-api.md

```diff
@@ -135,6 +136,27 @@ For more information on integrating Kong into custom Nginx configurations, see
 
 [Back to TOC](#table-of-contents)
 
+## Role Based Access Control ##
+
+<div class="alert alert-warning">
+  <strong>Enterprise-Only</strong> This feature is only available with an
+  Enterprise Subscription.
+</div>
+
+Enterprise users can configure role-based access control to secure access to the
+Admin API. RBAC allows for fine-grained control over resource access based on
+a model of user roles and permissions. Users are assigned to one or more roles,
+which each in turn possess one or more permissions granting or denying access
+to a particular resource. In this way, fine-grained control over specific Admin
+API resources can be enforced, while scaling to allow complex, case-specific
+uses.
+
+If you are not a Mashape Enterprise customer, you can inquire about our
+Enterprise offering by [contacting us](/enterprise).
```

Comment:
maybe `/enterprise/#demo-form` since we're talking about contacting us?

maybe also change form "contacting us" => contacting Mashape